tests/paramgrill.c

```diff
@@ -972,7 +985,29 @@ int main(int argc, const char** argv)
         if(!strcmp(argument,"--no-seed")) { g_noSeed = 1; continue; }
 
         /* Decode command (note : aggregated commands are allowed) */
-        if (argument[0]=='-') {
+        if (longCommandWArg(&argument, "--zstd=")) {
+            g_singleRun = 1;
```

Comment:
This means that `--zstd=#` acts as a command, with same impact as `-S#`.

This is not entirely obvious.
In `zstd`, `--zstd=#` only selects parameters for compression, but it doesn't presume what is done with them afterwards. It could be for compression, it could be for benchmark.

So at the very least, this behavior should be documented.
Use `usage_advanced()` for that.